src/app/app.component.html

```diff
@@ -1,13 +1,27 @@
-<div class="app">
-
-  <pfng-toast-notification-list [notifications]="notifications"
-                                (onActionSelect)="handleAction($event)"
-                                (onCloseSelect)="handleClose($event)"
-                                [showClose]="showClose">
-  </pfng-toast-notification-list>
-  <ng-template #brandTemplate>
-    <a [routerLink]=" ['/'] "
-       class="navbar-brand">
+<div class="app" [ngSwitch]="userService.checkToken()">
+
+    <!-- Does Not Have Valid Token -->
+
+    <ng-container *ngSwitchDefault>
+      <main class="container-fluid"
+            *ngIf="!loggedIn">
```

Comment:
is checkToken() and `loggedIn` the same thing?  Maybe this *ngIf is redundant?

Reply:
you are right, they are redundant but that's not new code added by this feature

Reply:
They are the same, but now we are instead making a call to the API to check to see if setup is required and setting the `firstTime` flag, so this `checkToken()` and `ngSwitch` will be going away. This is different because the user is likely to have a token at this point, as they'll be logged in.